inception/inception-security/src/main/java/de/tudarmstadt/ukp/inception/security/oauth/OAuth2AdapterImpl.java

```diff
@@ -148,6 +147,10 @@ private User fetchOrMaterializeUser(OAuth2UserRequest userRequest, OAuth2User us
         if (u != null) {
             denyAccessToDeactivatedUsers(u);
             denyAccessOfRealmsDoNotMatch(realm, u);
+
+            u.setRoles(OAuth2Utils.getOAuth2UserRoles(u, user.getAttribute("groups")));
```

Comment:
The claim should be configurable via the properties as well since this is not a well-defined name. Others might e.g. use a `roles` claim instead.